docs/source/sci_api_tutorial/PDBParser.rst

```diff
@@ -0,0 +1,174 @@
+==============================================
+PDBParser
+==============================================
+
+Briefs
+==============================================
+``PDBParser`` facilitates the conversion between PDB files and ``Structure`` objects. It primarily offers two functions: ``get_structure()`` for converting a PDB file into a ``Structure`` object, and ``save_structure()`` for performing the inverse operation.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/PDBParser_io.svg
+        :width: 100%
+        :alt: PDBParser_io
+
+PDBParser.get_structure()
+==============================================
+
+Input
+------------------------------------------------
+
+``path``
+    Specifies the file path of the PDB file.
+
+    .. admonition:: How to obtain
+
+        PDB files can be downloaded from the `Protein Data Bank <https://www.rcsb.org/>`_ or acquired from experimental results.
+
+Output
+------------------------------------------------
+
+``Structure()``
+    Returns the constructed ``Structure`` object.
+
+Arguments
+------------------------------------------------
+
+``path``
+    Path to the PDB file.
+    (See `PDBParser.get_structure() <#pdbparser-get-structure>`_ section)
+
+``model``
+    | Index of the model to select if the file contains multiple models.
+    | Determine if there are multiple models by looking for "MODEL {number}" and "ENDMDL" tags in the PDB file, or by using protein visualization tools such as `PyMOL <https://pymol.org>`_ or `UCSF Chimera <https://www.cgl.ucsf.edu/chimera/>`_.
+
+``add_solvent_list``
+    | List of additional solvent names. If solvents other than water are present, include them here.
+    | For example, ``add_solvent_list=["DMS", "ACT"]``
+    | Solvents are recognized by names in these lists within non-polypeptide chains:
+    | ``chem.RD_SOLVENT_LIST + add_solvent_list``
+    | ``RD_SOLVENT_LIST`` includes common water aliases such as HOH and WAT. (``RD_SOLVENT_LIST: List[str] = ["HOH", "WAT"]``)
+
+``add_ligand_list``
```

Comment:
It will only influence the RD_NON_SOLVENT_LIST. change the desc to what we discussed.

Reply:
Done